tests/ipython/test_ipython.py

```diff
@@ -62,21 +63,13 @@ def my_register_pipeline():
             return_value=my_register_pipeline,
         )
         mocker.patch("kedro.framework.startup.configure_project")
```

Comment:
```suggestion
        mocker.patch("kedro.ipython.configure_project")
```

This is the wrong patch that causing most of the confusion.

Reply:
Ahh, if I change this then can I remove the `no pragma` and the tests pass?

Reply:
Let's address this altogether at #1839